upgrade/UPGRADE-v9.1.0-dev.md

```diff
@@ -68,3 +68,6 @@ There you can find links to upgrade notes for other versions too.
 
 - fix displaying '+' sign in product filter ([#2023](https://github.com/shopsys/shopsys/pull/2023))
     - see #project-base-diff to update your project
+
+- remove setting domain locale in `CartFacadeTest` ([#2037](https://github.com/shopsys/shopsys/pull/2037))
```

Comment:
I would mention that in tests is used different class as an alias for `Shopsys\FrameworkBundle\Model\Localization\TranslatableListener`. 
But I'm ok with the current state.

Reply:
Updated